### PR TITLE
PERF-649: Configure alembic

### PR DESCRIPTION
**Pull request by andrebco**
@olel-gyg and @olga-andreeva-gyg, please validate that the configuration is also working on your side. Please feel free for adjusting anything you want.

### CODE REVIEW

File: app/alembic/env.py [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
+import sys
 
+# Appending on path
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), "../")))
 
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
 
+# get_url function
```
Comment (cfalcondair) [inline app/alembic/env.py]: Is this comment needed?

File: app/alembic/env.py [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
+def get_url():
```
Comment (cfalcondair) [inline app/alembic/env.py]: could this be just `url()`? Mind you I'm not familiar with the python syntax...

File: app/alembic/env.py [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-
-from flask import current_app
-from app.config import Config
-
-config.set_main_option('sqlalchemy.url', Config.SQLALCHEMY_DATABASE_URI)
-target_metadata = current_app.extensions['migrate'].db.metadata
+from app.ad_templates.models import *
```
Comment (cfalcondair) [inline app/alembic/env.py]: Would explicit imports help namespace management here?

File: app/alembic/env.py [hunk 4]
```diff
@@ -58,32 +65,19 @@ def run_migrations_online():
...
-    connection = engine.connect()
-    context.configure(connection=connection,
-                      target_metadata=target_metadata,
-                      process_revision_directives=process_revision_directives,
-                      **current_app.extensions['migrate'].configure_args)
-
-    try:
         with context.begin_transaction():
             context.run_migrations()
-    finally:
```
Comment (cfalcondair) [inline app/alembic/env.py]: Just wanna confirm that we don't need to close the connection here?

File: app/alembic/env.py (hunk 1 above)
Comment (andrebco) [reply app/alembic/env.py]: No. ;-) Removing in 3, 2, 1...

File: app/alembic/env.py (hunk 2 above)
Comment (andrebco) [reply app/alembic/env.py]: it could be, but its kind of a convention on alembic config.

File: app/alembic/env.py (hunk 3 above)
Comment (andrebco) [reply app/alembic/env.py]: Totally agree, but on this case it was in purpose so we have new models being automatically generated.

File: app/alembic/env.py (hunk 4 above)
Comment (andrebco) [reply app/alembic/env.py]: We are using `with`on the previous contexts.

**Pull request closed without merging**